### PR TITLE
Blitzshell Rework

### PR DESCRIPTION
**Pull request by Jamini**
## About The Pull Request

This PR completely reworks Blitzshell into a new antagonist type: Rogue Drones
- Rogue Drones are drones that are no longer synced to the drone network/master controller. Over time their lawset has been corrupted  into a perversion of what it was once.
- Rogue Drones are no longer contract antagonists, dramatically lowering their available Telecrystals
- Rogue Drones no longer have most blitzshell modules. Only the nanorepair, uplink, and smoke  smoke generator (Uplink only) remain.
- Rogue Drones start with a cyborg Hatton and Taser. Allowing for limited (nonelthal) self-defense and brute-forcing access to certain areas. be wary, the Hatton stuns you on use for several seconds!
-  Rogue Drones have a full maintenance drone loadout, allowing them to hack, sabotage, or redesign to their twisted synthetic desires!
- Rogue Drones have NO ship access, and no means to acquire more. They can only access what they can physically reach via ventcrawl.
- Rogue Drones are able to listen in on the drone and silicon channels.
- Rogue Drones, like all drones, are not capable of speech nor self-repair. However they are able to emote.
- Drones start at 90hp, and are able to augment up to 210HP with their starting TC.
- Drones are able to fabricate additional drones at a high cost in crystals. Note that there MUST be a free and willing ghost to join as the new drone, or else the TC are wasted!
- Rogue Drones have corrupted laws that encourage them to interfere with, harm, and sabotage the crew! Be wary, the adorable little hats you know and love might turn around and tase you in the back if you aren't careful!
- Rogue Drones, like their regular counterparts, can hide under objects and tables with the Hide command. That said, bullets will not pass over them. Spray and play baby!
- Rogue Drones do not take agony damage like all synthetics. However they are vulnerable to EMPs and are slow to compensate! Pick your battles carefully as a drone, because escape may not be an option!

image

image

## Why It's Good For The Game

A re-imagining of an old, disabled antagonist type that should help us keep things fresh and interesting. Rogue drones are less focused on carnage, and more focused on sabotage and harassment.

## Changelog
```changelog
tweak: Adjusted several ghostrole prompts to be less 'Autistic'
fix:: Fixed a typo in antagonist welcome text.
fix: Fixed spelling on the Industrial Chemical Dispenser circuit
add: Completely re-imagined Blitzshell into a less murderboner antagonist type.
```

### CODE REVIEW

File: zzzz_modular_occulus/code/modules/stashes/stashes.dm [hunk 1]
```diff
@@ -90,7 +90,7 @@
 	Is the tale of Jayme Dawson and his crew <br>\
 	Yes, the tale of Dawson's Christian and her crew.<br>\
 	<br>\
-	D%"
+	%D"
```
Comment (Michiyamenotehifunana) [inline zzzz_modular_occulus/code/modules/stashes/stashes.dm:93]: REEEEE THIS IS THE THIRD TIME WE'VE DONE THIS

**Review by Michiyamenotehifunana — CHANGES_REQUESTED** [review]
I know it literally doesn't matter and git should handle it gracefully but there's more of that same typo fix that already got merged.

**Review by Michiyamenotehifunana — DISMISSED** [review]
Nothing looks out of the ordinary, it's gonna be interesting to see how this pans out on the live server.

**Review by zackerycodestoo — APPROVED** [review]
Proxy approval for Tori, blame Tori, all Tori's fault

**Pull request merged**